maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractRewritePomsPhase.java

```diff
@@ -96,7 +96,7 @@ public abstract class AbstractRewritePomsPhase extends AbstractReleasePhase impl
      * Regular expression pattern matching Maven expressions (i.e. references to Maven properties).
      * The first group selects the property name the expression refers to.
      */
-    private static final Pattern EXPRESSION_PATTERN = Pattern.compile("\\$\\{(.+)\\}");
+    private static final Pattern EXPRESSION_PATTERN = Pattern.compile("\\$\\{(.+?)\\}");
```

Comment:
Causes [MRELEASE-1151](https://issues.apache.org/jira/browse/MRELEASE-1151). This caused the weekly Jenkins core release to fail on July 2, 2024. We have downgraded Maven Release Plugin to 3.0.1 in order to restore stability to our releases.

Reply:
As a workaround you could try to add configuration property `<arguments>-Dchangelist=</arguments>` for the `maven-release-plugin`

Reply:
> Causes [MRELEASE-1151](https://issues.apache.org/jira/browse/MRELEASE-1151).

As explained in https://github.com/apache/maven-release/pull/202#pullrequestreview-2169941878, https://github.com/apache/maven-release/pull/202 seems to resolves this issue (with one exception), and I think this PR should not have been released without also including https://github.com/apache/maven-release/pull/202. I would recommend that the next release either include https://github.com/apache/maven-release/pull/202 or revert this PR.